src/main/scala/viper/gobra/translator/implementations/translator/MethodsImpl.scala

```diff
@@ -37,6 +37,10 @@ class MethodsImpl extends Methods {
     for {
       pres <- sequence((vRecvPres ++ vArgPres) ++ x.pres.map(ctx.ass.precondition(_)(ctx)))
       posts <- sequence(vResultPosts ++ x.posts.map(ctx.ass.postcondition(_)(ctx)))
+      terminationMeasure = x.terminationMeasure.isEmpty match {
+        case true => Seq.empty
+        case false => sequence(x.terminationMeasure.map(ctx.ass.terminationMeasure(_)(ctx))).res
+      }
```

Comment:
Can this be simplified to `terminationMeasure <- sequence(x.terminationMeasure.map(ctx.ass.terminationMeasure(_)(ctx)))`?